descriptors/ModuleDescriptor-template.json

```diff
@@ -38,7 +38,10 @@
             "modperms.orders.item.post",
             "orders-storage.titles.item.get",
             "orders-storage.configuration.prefixes.collection.get",
-            "orders-storage.configuration.suffixes.collection.get"
+            "orders-storage.configuration.suffixes.collection.get",
+            "finance.funds.item.get",
+            "finance.transactions.collection.get",
+            "finance.transactions.batch"
```

Comment:
we missed to add permissions to POST order in scope of https://folio-org.atlassian.net/browse/MODORDERS-859